docs/user-guide/plotting.ipynb

```diff
@@ -522,27 +501,28 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "N = 2000\n",
-    "M = 1000\n",
-    "xx = np.arange(N, dtype=np.float64)\n",
-    "yy = np.arange(M, dtype=np.float64)\n",
-    "x, y = np.meshgrid(xx, yy)\n",
-    "b = N/1000.0\n",
-    "c = M/100.0\n",
-    "r = np.sqrt(((x-c)/b)**2 + ((y-c)/b)**2)\n",
-    "a = np.sin(r)\n",
-    "large = sc.Dataset()\n",
-    "large.coords[Dim.X] = sc.Variable([Dim.X], values=xx, unit=sc.units.m)\n",
-    "large.coords[Dim.Y] = sc.Variable([Dim.Y], values=yy, unit=sc.units.m)\n",
-    "large['Signal'] = sc.Variable([Dim.Y, Dim.X], values=a, unit=sc.units.counts)\n",
-    "plot(large)"
+    "N = 40\n",
+    "M = 5\n",
+    "x = np.arange(N).astype(np.float64)\n",
+    "b = 0.5 * N\n",
+    "a = 4.0*np.random.rand(M, N)\n",
+    "a[2, :] = np.abs(10.0 * np.cos((x-b)*2.0/b))\n",
+    "d2 = sc.Dataset()\n",
+    "d2.coords[Dim.Tof] = sc.Variable([Dim.Tof], values=x, unit=sc.units.us)\n",
+    "d2.coords[Dim.X] = sc.Variable([Dim.X], values=np.arange(M).astype(np.float64),\n",
+    "                               unit=sc.units.m)\n",
+    "d2['sample'] = sc.Variable([Dim.X, Dim.Tof], values=a,\n",
+    "                           variances=0.1*np.random.rand(M, N))\n",
+    "plot(d2)\n",
+    "plot(d2, collapse=Dim.Tof)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The size limit (number of pixels) above which a 2D image will be rasterized can be changed by setting:"
+    "### Image aspect ratio\n",
+    "By default, the aspect ratio of 2D images is preserved. You can choose to stretch the image to the figure size via the `aspect` keyword argument:"
```

Comment:
I think we should not make this the default. Except for examples in documentation, I believe this default would almost always be wrong, because dimension extents typically differ drastically, resulting in unreadable plots?